common/djangoapps/pipeline_mako/templates/static_content.html

```diff
@@ -96,6 +96,7 @@
         try:
             return Template("""
                 {% load render_bundle from webpack_loader %}
+                {% render_bundle "commons" %}
```

Comment:
@ormsbee Is there an issue for pages with multiple Webpack sections? It seems that this will load the commons bundle several times.